libbeat/logp/logger.go

```diff
@@ -203,6 +205,14 @@ func (l *Logger) DPanicw(msg string, keysAndValues ...interface{}) {
 	l.sugar.DPanicw(msg, keysAndValues...)
 }
 
+// Recover stops a panicking goroutine and logs an Error.
+func (l *Logger) Recover(msg string) {
```

Comment:
It doesn't look like this is called anywhere yet (including tests), should it be? Or is it just for use in followups?

Reply:
It is for the followups.